app_dev_docker.yaml

```diff
@@ -0,0 +1,145 @@
+  expiration: "0"
+# /dist is the build output folder for angular.
+- url: /dist
+  static_dir: dist/
+  secure: always
+  expiration: "0"
+- url: /assets
+  static_dir: assets
+  secure: always
+  http_headers:
+    # This is replaced by a specific origin when doing a deployment.
+    Access-Control-Allow-Origin: "*"
+  expiration: "0"
+# Serve CSS files under core/templates.
+# This regex allows us to recursively serve CSS files.
+# "\1" inserts text captured by the capture group in the URL pattern.
+- url: /templates/(.*\.(css))$
+  static_files: core/templates/\1
+  upload: core/templates/(.*\.(css))$
+  secure: always
+  expiration: "0"
+- url: /templates/(.*\.(html))$
+  static_files: core/templates/\1
+  upload: core/templates/(.*\.(html))$
+  secure: always
+  expiration: "0"
+- url: /third_party/generated
+  static_dir: third_party/generated
+  secure: always
+  http_headers:
+    # This is replaced by a specific origin when doing a deployment.
+    Access-Control-Allow-Origin: "*"
+  expiration: "0"
+- url: /extensions/interactions/(.*)/static/(.*\.(css|png))
+  static_files: extensions/interactions/\1/static/\2
+  upload: extensions/interactions/(.*)/static/(.*\.(css|png))
+  secure: always
+  expiration: "0"
+- url: /extensions/(interactions|rich_text_components)/(.*)/directives/(.*\.(html))
+  static_files: extensions/\1/\2/directives/\3
+  upload: extensions/(interactions|rich_text_components)/(.*)/directives/(.*\.(html))
+  secure: always
+  expiration: "0"
+# Serve PNG images for interactions, rich_text_components, objects, classifiers,
+# and custom ckeditor plugins under extensions in dev mode. This regex allows
+# us to recursively serve PNG images scripts under the five specified
+# directories. "\1" and "\2" insert capture groups from the url pattern.
+- url: /extensions/(interactions|rich_text_components|objects|classifiers|ckeditor_plugins)/(.*\.(png))$
+  static_files: extensions/\1/\2
+  upload: extensions/(interactions|rich_text_components|objects|classifiers|ckeditor_plugins)/(.*\.(png))$
+  secure: always
+  expiration: "0"
+- url: /extensions/ckeditor_plugins/(.*\.(js))$
+  static_files: extensions/ckeditor_plugins/\1
+  upload: extensions/ckeditor_plugins/(.*\.(js))$
+  secure: always
+  expiration: "0"
+- url: /extensions/visualizations/(.*\.html)
+  static_files: extensions/visualizations/\1
+  upload: extensions/visualizations/(.*\.html)
+  secure: always
+  expiration: "0"
+- url: /extensions/objects/templates/(.*\.html)
+  static_files: extensions/objects/templates/\1
+  upload: extensions/objects/templates/(.*\.html)
+  secure: always
+  expiration: "0"
+
+# This is an access connector that connects to the Google Cloud Redis instance
+# used for memory caching. Please replace PROJECT_ID with the correct project id
+# for oppia on the production server.
+vpc_access_connector:
+  name: projects/PROJECT_ID/locations/us-central1/connectors/oppia-redis-connector-1
+
+env_variables:
+  oppia_is_dockerized: "true"
```

Comment:
Make it all caps and add a comment explaining why this need to be added here.

Reply:
done

Reply:
confirmed that this is done